pkg/controller/ground.go

```diff
@@ -610,8 +610,8 @@ func (op *GroundControl) createKluster(kluster *v1.Kluster) error {
 	}
 
 	//contains unamibious characters for generic random passwords
-	var randomPasswordChars = []rune("abcdefghjkmnpqrstuvwxABCDEFGHJKLMNPQRSTUVWX23456789")
-	klusterSecret.NodePassword, err = goutils.Random(12, 0, 0, true, true, randomPasswordChars...)
+	var randomPasswordChars = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890")
```

Comment:
we specifically left out ambiguous characters initially to avoid problems reading a password e.g. '1' and 'l' can look similar depending on the font type.